docs/src/howto-guides/002_installation.md

```diff
@@ -39,11 +39,12 @@ When all dependencies are in place you can proceed with installation of `thin-ed
 
 If you already have `thin-edge.io` on your device to upgrade `thin-edge.io` follow the steps below, there is no need to remove old version.
 
-> Note: To successfully upgrade `thin-edge.io` all thin-edge.io components should be stopped, eg: `tedge-mapper` and `tedge-agent`:
+> Note: To successfully upgrade `thin-edge.io` all thin-edge.io components must be stopped before upgrade, and started again afterwards. The simpler is to use the commands `tedge disconnect <cloud>` and `tedge connect <cloud>`. Example:
```

Comment:
I really support this change. As of now, a user needs to stop maximum 4 services to upgrade all packages. `tedge disconnect` does all by just one command. It's more user-friendly. It is meaningful that we give this option to users in our user-guide.